### ISSUE TITLE
Website seems hacked

### ISSUE THREAD

**Issue by TheDigitalNinja** [camping/camping#97]

**Comment by sfcgeorge** [comment]
:( Seems so. ruby-camping.com still works but I assume is an outdated one. camping.io is dead. whywentcamping.com is also dead. As the camping.io site is a gh-pages one, I think the CNAME file could just be deleted then it will be available again at camping.github.io/camping.io

**Comment by pygy** [comment]
I deleted the CNAME files (I wanted to send a PR, I didn't remember I had write access here, ... click, click... poof the changes are live... Errr... well, it didn't hurt :-)

@judofyr The camping.io URL just gave my browser cancer... It flooded the history with trash and ultimately redirected to an adult site... 

I can't edit the GH settings to point to camping.github.io/camping.io rather than the now squatted camping.io

**Comment by DaveEveritt** [comment]
At least I've just removed the spammy link next to the GitHub title. One of the links on camping.github.io/camping.io goes to the spammy site, so we need remove that link from the website and check through for others like *.camping.io before making it public again.

**Comment by topofocus** [comment]
Hi Altogether
just typed "camping.io" into the browser .... redirection to 
https://www.auskunft.de/?rt=y&utm_source=redirect-com&uclid=wUMA7P9VBV6OG2JG1H81KQN2&tsrcid=cb10db33-3ee2-4fa0-be10-d6d18d2dc772 
occured

http://camping.github.io/camping.io/  seems to work,  looks very, very outdated still

**Comment by DaveEveritt** [comment]
"camping.io" was left to rot as no-one was free to maintain it :-(

Also, no-one has volunteered to update the github site… repo was to be here: https://github.com/camping/camping.io

and the "book" is still here: https://github.com/camping/camping/blob/master/book/01_introduction.md

**Comment by DaveEveritt** [comment]
> Does anybody here know who controls the DNS for camping.io? I'd love to fix the issues with the domain.

Please also take a look at this old issue about the disparate Camping websites: #60

**Comment by karloscarweber** [comment]
@DaveEveritt Yeah it seems that we don't have access to camping.io. I've been trying to buy the domain but can't find a way to contact the current owner. Everything I tried didn't work. I think we should make a brand new website from scratch, running from GitHub pages. I have the domain **rubycamping.org** and it think that would work well.

**Comment by topofocus** [comment]
I like that!

**Comment by DaveEveritt** [comment]
The repo for the last website http://camping.github.io/camping.io/ is still a good basis, it's minimal, static (future-proof), has the basic info and (most importantly after _why disappeared, which is when all this started!) a _why-like design. Just needs the menu updating and to be made properly responsive with updated code.

**Comment by karloscarweber** [comment]
I'll work on updating that site then. But some things are just a bit too dated to update reliably. Modern web-design has kinda "_moved on_". I'll first go through and make sure the links don't lead to spam or dead ends.

**Comment by DaveEveritt** [comment]
> I'll work on updating that site then. But some things are just a bit too dated to update reliably. Modern web-design has kinda "_moved on_". I'll first go through and make sure the links don't lead to spam or dead ends.

good start - agreed about the 'moved on'—although the markup's pretty solid, at least it could be mobile-first/responsive. I've asked one of the old team to add me as an admin to the Camping org, so at least I can clean the place up :-)

**Comment by DaveEveritt** [comment]
https://github.com/camping/whywentcamping.com could become the new repo for "rubycamping.org" - if agreed, I'll rename the repo ("whywentcamping.com" spams to Apple Music) and you could extract the content from the existing http://camping.github.io/camping.io/ there and anything useful in the whywentcamping.com repo - be good to stick with a Ruby-based build but keep the site simple and static—for Ruby, [Nanoc](https://nanoc.app) works well and is free from any opinion structures or boilerplate, and [the dev is really nice](https://github.com/ddfreyne) :-)

**Comment by karloscarweber** [comment]
@DaveEveritt I like all of this but why not rename the **camping.io** repo to the new domain? Is it to preserve the git history? Because I gotta tell you, I have learned so much about camping from reading everything in the old commits.

**Comment by DaveEveritt** [comment]
Good idea. How about this:

1. leave the camping.io repo for now to preserve its [GitHub pages](http://camping.github.io/camping.io/) site,
2. create a new repo "rubycamping.org" to set up the new site and GitHub pages, as a *clone of camping.io* that retains the gh-pages branch intact, then…
3. remove the camping.io repo once all the content in the [gh-pages branch](https://github.com/camping/camping.io/tree/gh-pages) is moved over to "rubycamping.org"?

I like the idea of preserving the history, hence starting "rubycamping.org" as a clone of camping.io, also retaining the gh-pages branch intact as a historical branch. If I've missed anything or got too complicated, please say!

BTW interesting that [whywentcamping.com](https://github.com/camping/whywentcamping.com) uses [Webby](https://github.com/TwP/webby) which is no longer maintained, but recommends Nanoc instead. Happy to pitch in with the Nanoc boilerplate if necessary.

**Comment by karloscarweber** [comment]
@DaveEveritt Gonna enact this plan outlined above sometime this summer. Rubycamping.org right now is live and is just a landing page.
